migrations/2021-06-15-151515_circulating_supply_table/up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE circulating_supply
```

Comment:
We already have a number of tables in the database, so I suggest we have some prefix in the naming for such precomputed tables. I may suggest `stats__circulating_supply` (please, suggest a better one if you have)

Reply:
Thoughts, ideas about prefix:
aggregation
aggregations
views
analytics
stats
statistics
cache
2 prefixes?

Come up with `aggregated__circulating_supply`